C2.md

```diff
@@ -63,11 +63,11 @@ be requested via C2 DESCRIBE manifest command.
 	# specify classes for the AST response
 	nifi.c2.root.classes=DeviceInfoNode,AgentInformation,FlowInformation
 	
-	# specify C2 protocol -- default is CoapProtocol if not specified
-	#nifi.c2.agent.protocol.class=CoapProtocol
-	# may also use MQTT or REST
-	# nifi.c2.agent.protocol.class=MQTTC2Protocol
+	# specify C2 protocol -- default is RESTSender if not specified
 	nifi.c2.agent.protocol.class=RESTSender
+	# may also use MQTT or CoapProtocol
+	# nifi.c2.agent.protocol.class=MQTTC2Protocol
```

Comment:
Do we have MQTT tested or at least documented anywhere?
Honestly, I have never tried it.

Reply:
I couldn't find it being tested anywhere, there is some sporadic mention of it in the docs, but nothing that would indicate that it has ever been used, we could investigate it in a separate PR if we would like to remove it, or revive it